addons/website/views/res_config_settings_views.xml

```diff
@@ -59,7 +59,7 @@
                                     <label class="col-lg-3" string="Company" for="website_company_id"/>
                                     <field name="website_company_id" required="website_id" title="The company this website belongs to"/>
                                 </div>
-                                <div class="row mt8">
+                                <div class="row mt8" id="favicon_details">
```

Comment:
Tz on website settings which would just be used by rental app?
Isn't this something to be moved inside the rental app config itself then?
Will be confusing imo, to be removed, moved or explained, like that it's a setting and one has no clue what it does

![image](https://github.com/odoo/odoo/assets/30048408/164c42c4-8b7a-43be-848f-5394c07d653b)
cc @pierre-pde

Reply:
Hello @rdeodoo :wave: 

Thanks for your comment here :) I don't know about Appointment, but I'll let them handle their use case. We add the timezone in `web` and it shouldn't impact the current state of Appointment as it's optional and fallback on the default tz.

As for the placement of the timezone, I do agree that it could be confusing if it's only related to rental. I'll let @dalaOdoo check.

Reply:
> I don't know about Appointment, but I'll let them handle their use case.

I meant it in the way: worth a look to see if they did not already find a way to workaround the issue without having to change base.
But again, I am speaking without really knowing / having check anything.

Reply:
yes we should mode this in rent online settings for the moment
![image](https://github.com/odoo/odoo/assets/100136841/fdd598bf-47a3-4e02-ae01-a7c0997eec68)